rust/chains/abacus-ethereum/src/trait_builder.rs

```diff
@@ -25,8 +25,9 @@ const METRICS_SCRAPE_INTERVAL: Duration = Duration::from_secs(60);
 const HTTP_CLIENT_TIMEOUT: Duration = Duration::from_secs(60);
 
 /// A trait for dynamic trait creation with provider initialization.
+
 #[async_trait]
-pub trait MakeableWithProvider {
+pub trait MakeableWithProvider: Sync {
```

Comment:
Curious what changed to make this needed now, but not really worth thinking about too long since we will probably be replacing this trait soon.